python/utilities/submit_chain/submit_chain.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+"""Submits a chain to a list of logs."""
+
+import base64
+import hashlib
+import sys
+
+import json
+import logging
+import gflags
+
+from ct.client import log_client
+from ct.crypto import cert
+from ct.crypto import error
+from ct.crypto import pem
+from ct.crypto import verify
+from ct.proto import client_pb2
+from ct.serialization import tls_message
+
+FLAGS = gflags.FLAGS
+
+gflags.DEFINE_string("log_list", None, "List of logs to submit to.")
```

Comment:
And... reading the code further down, I find out that this is really a file. I'm no longer sure what this should be, I'm only sure at least the doc string should be improved. :wink: